cfr.yaml

```diff
@@ -719,3 +719,60 @@
     notification_text:
       string_id: cfr-doorhanger-feature-notification
     category: cfrFeatures
+- id: HOMEPAGE_REMEDIATION
+  groups:
+    - cfr
+  frequency:
+    lifetime: 3
+  # Has google set as the homepage, has CFR features enabled, has profile
+  # older than 6 months
+  targeting: "!homePageSettings.isDefault && homePageSettings.isCustomUrl && homePageSettings.urls[.host == 'google.com']|length > 0 && visitsCount >= 3 && userPrefs.cfrFeatures && currentDate|date - profileAgeCreated > 15770000000 && firefoxVersion > 82 && localeLanguageCode == 'en'"
+  template: cfr_doorhanger
+  content:
+    bucket_id: HOMEPAGE_REMEDIATION
```

Comment:
Since this CFR is only targetting users who visit google.com in the release, per CFR privacy requirement, we have to use the generic `bucket_id` like the one (`CFR_M1`) used by the other addon CFRs.